kajas/src/app/edit-artwork/edit-artwork.component.html

```diff
@@ -0,0 +1,80 @@
+<app-header></app-header>
+
+<div class="edit-artwork">
+  <div class="modal">
+    <div class="modal-content">
+      <h1>Edit Artwork</h1>
+      <form [formGroup]="artworkForm" (ngSubmit)="save()">
+        <div class="content">
+          <div class="left-column">
+            <div class="form-group">
+              <div class="image-upload">
+                <div class="artwork-pic-group">
+                  <label class="artwork-pic-label">
+                    <img
+                      id="artwork-placeholder"
+                      [src]="getAbsoluteUrl(imageUrl) || 
+                        '../../assets/icons/placeholder_image.png'"
+                      alt="Artwork Placeholder"
+                    />
+                  </label>
+                </div>
+              </div>
+            </div>
+          </div>
+          <hr class="divider" />
+          <div class="right-column">
+            <div class="form-group">
+              <label for="title">Artwork Title:</label>
+              <input
+                type="text"
+                id="title"
+                formControlName="title"
+                [attr.placeholder]="titlePlaceholder"
+              />
+              @if(artworkForm.get('title').invalid &&
```

Comment:
@if(artworkForm.get('title').invalid &&
              artworkForm.get('title').touched) {
              <div class="error-message">{{ getErrorMessage("title") }}</div>
              }
### For template control flow, observe proper indentation.
[https://docs.google.com/presentation/d/1Mdcq3WcHksBlbCvxi6wx5vajn4v8LGlJRnBIu3x4fvw/edit#slide=id.g270ee9174f4_0_15](url)